app/models/form.rb

```diff
@@ -9,7 +9,7 @@ def initialize(form_path, start_year = "", sections_in_form = [], type = "lettin
       @end_date = if start_year && start_year.to_i > 2022
                     Time.zone.local(start_year + 1, 6, 9)
                   else
-                    Time.zone.local(start_year + 1, 6, 7)
+                    Time.zone.local(start_year + 1, 8, 7)
```

Comment:
I think we agreed end of collection year as we're expecting this to be in september when Rachel confirms

Reply:
I wasn't clear on what date it was but I feel like within two months we need to have a clearer idea of what different end dates are and how they are implemented.
I'm slightly uneasy about putting it so far in the future as it makes it easier to just leave it working and forget all about this.
If we're leaving it like this for any length of time, tests need writing etc to cover it so if we don't have better solutions conceptually we should at least make a PR for writing tests etc and then this can be updated if we wish

Reply:
Also I'm imagining we should change this for years after 2022/23 too (line 10), so we can just replace the whole if/else with a single date

Reply:
your first point is fine by me 👍, we do need to update this either way